brownfield_django/mixins.py

```diff
@@ -4,8 +4,7 @@
 
 from django.contrib.auth.decorators import login_required, user_passes_test
 from django.contrib.auth.models import User
-from django.http.response import HttpResponseNotAllowed, HttpResponse, \
-    HttpResponseForbidden
+from django.http.response import HttpResponse, HttpResponseForbidden
 from django.utils.decorators import method_decorator
 from django.views.decorators.csrf import csrf_exempt
 
```

Comment:
Same thing here, we can test like this: https://github.com/ccnmtl/writlarge/blob/24b9bc28b9b2e7643221c4bd133806271724d4c1/writlarge/main/mixins.py#L17

Reply:
Done